src/test/kotlin/de/europace/gradle/docker/publish/DockerPublishPluginIntegrationTest.kt

```diff
@@ -59,10 +59,50 @@ class DockerPublishPluginIntegrationTest : FreeSpec() {
         :publishImage SKIPPED
         :rmiLocalImage SKIPPED
         """.trimIndent()
-      result.output shouldStartWith expectedOutput
+      result.output.normalizedLineSeparators() shouldStartWith expectedOutput.normalizedLineSeparators()
       result.output shouldContain "BUILD SUCCESSFUL"
     }
 
+    "publishImage should lookup the authConfig without errors" {
+      buildFile.writeText(
+          """
+          plugins {
+            id("$PLUGIN_ID")
+          }
+
+          dockerPublish {
+            organisation.set("foo")
+          }
+
+          tasks {
+            create("bootJar") {
+              doFirst {
+                logger.lifecycle("Would now create jar file")
+              }
+            }
+            withType<de.gesellix.gradle.docker.tasks.DockerPushTask>().matching { it.name == "publishImage" }.configureEach {
+              doFirst {
+                logger.lifecycle("image: ${"$"}{repositoryName.get()}, authConfig: ${"$"}{if (authConfig.get() == null) {"missing"} else {"found"}}")
```

Comment:
The `${"$"}` workaround of an escaped dollar sign in multiline strings is taken from https://youtrack.jetbrains.com/issue/KT-2425.